vm/vm.go

```diff
@@ -345,25 +408,5 @@ func (vm *Engine) Execute() error {
 		}
 	}
 
-	// Now that we know we're not dealing with a genesis state transition,
-	// we'll map our set of asset inputs and outputs to the 1-input 1-output
-	// virtual transaction.
-	virtualTx, inputTree, err := tapscript.VirtualTx(
```

Comment:
Still very important. 

See commetn above, we can retain this by leaning on Bitcoin for the sighash side of things.